book/multi-arch-controller.md

```diff
@@ -0,0 +1,114 @@
+# Multi Arch Controller
```

Comment:
Would we gain any benefit if we abstract this controller away from builds specifically? In essence, it is a remote environment provisioner. In the implementation of this controller, we are exposing the fact that this is implemented for builds because of the use of `build-dir`. We are also exposing the fact that this is created for the creation of environments with multiple architectures.

This request is effectively one to implement against the Environment API, but I suspect that this would also prevent the use of the novel scheduling approach using the secrets. Maybe the Environment API is _too_ abstracted, but the approach can be slightly abstracted (i.e. renaming `build-dir` to `user-dir`).

Reply:
I can rename the directory, it can be used for any task that requires a virtual machine to be provisioned, its not just for builds.